src/content/troubleshooting/_index.en.md

```diff
@@ -217,11 +217,72 @@ Status:
 Events:  <none>
 ```
 
-Note that for a headless Service, the `Ips` field will contain the list of associated Endpoint IPs.
+For headless Service, you need to check `EndpointSlice` resource.
```

Comment:
This is because for ClusterIP currently we don't need to check anything. so L194-217 is continuation of what we have at L189. This will change and we'll revisit this once Aswin's patch to use endpointslice for ClusterIP is merged.